src/api/forms/components/FormRow.vue

```diff
@@ -120,9 +120,13 @@ export default {
         valid = regex.test(data.value);
       }
 
-      const validate = data.model.validate;
-      if (valid && validate) {
-        valid = validate(data);
+      if (data.isValid !== undefined && data.isValid !== null && !data.isValid) {
```

Comment:
I created an 'isValid' property on 'data' to work around the broken 'data.model.validate' logic below.